### DISCUSSION TITLE
What are the restrictions of hotkeys without an activation keybind?

### DISCUSSION THREAD

**Discussion by GaussianWonder** [FancyWM/fancywm#240] [Q&A]
1. What are the restrictions of hotkeys without an activation keybind?

    I want to be able to hold down the `meta` key and press `1`, `2`, `[`, `{` (...) and have the keybinds still function. Doing this with keychords is somewhat slower than I am used to on linux WMs.

I am aware of windows having hard coded and unchangeable keybinds that start with the windows key. I tried swapping my `meta` key with `F13`-`F24` keys, but the global hotkeys cannot be set.

2. Is it a windows restriction? Does it programmatically not allow you to set shortcuts without the **Shift**, **Control**, **Alt** or **Windows** as part of the shortcut?

**Comment by veselink1** [comment]
Before FancyWM 2.12, it used to use the Win32 RegisterHotkey function, and inherited it's restrictions.

https://github.com/FancyWM/fancywm/commit/64297c5b34acc13ea6b57d968d06f24e95b3df3d

As of 2.12, it now uses its own hot key implementation, which reuses the LowLevelHotkey used by the ActivationHotkey.

The new implementation technically allows you to bind what you want, but I believe the Settings UI will not capture the Windows key when configuring (as it does not use the LowLevel implementation during input capture yet). You can try editing the settings.json.
